doc/tutorial.rst

```diff
@@ -546,38 +546,79 @@ example is:
 
   type t = string <ocaml valid="fun s -> String.length s >= 8"> option
 
-``atdgen -v`` will produce something equivalent to the following
-implementation:
+As we can seefrom this example, the validation function is specified using the
+annotation ``<ocaml valid=p>``, where ``p`` is a predicate ``p : t -> bool``,
+returning ``true`` when the value of type ``t`` is valid and ``false``
+otherwise.
```

Comment:
2 things here:
* seefrom -> see from
* `<ocaml valid=p>` would be clearer as `<ocaml valid="p">` I think.